pandas-stubs/core/arrays/sparse/array.pyi

```diff
@@ -65,15 +103,24 @@ class SparseArray(ExtensionArray):
     def __getitem__(  # ty: ignore[invalid-method-override]
         self, key: SequenceIndexer | tuple[int | ellipsis, ...]
     ) -> Self: ...
-    def copy(self): ...
-    def map(self, mapper): ...
-    def to_dense(self): ...
-    def nonzero(self): ...
-    def all(self, axis=..., *args: Any, **kwargs: Any): ...
-    def any(self, axis: int = ..., *args: Any, **kwargs: Any): ...
-    def sum(self, axis: int = 0, *args: Any, **kwargs: Any): ...
-    def cumsum(self, axis: int = ..., *args: Any, **kwargs: Any): ...
-    def mean(self, axis: int = ..., *args: Any, **kwargs: Any): ...
+    def copy(self) -> Self: ...
+    def map(
+        self, mapper: dict[Any, Any] | Series[Any] | Callable[..., Any]
+    ) -> Self: ...
```

Comment:
missing `na_action` argument?

Reply:
Indeed I did not fix missing args, just documenting to clear the warnings (there is no docs for any of those methods so had to go in the source code).